generator/dart/packages/google_cloud_common/lib/common.dart

```diff
@@ -22,16 +23,17 @@ abstract class Jsonable {
   Object toJson();
 }
 
-abstract class CloudMessage {}
+abstract class CloudMessage implements Jsonable {}
```

Comment:
Instead of `CloudMessage` (and `CloudEnum`), I could use `RpcMessage` and `RpcEnum` 🤷

Reply:
They're worth trying! The identifiers we definitely can't use are https://dart.dev/language/keywords. We can try Message and Enum for now and pivot if they become problematic.